rdf-abac-core/src/main/java/io/telicent/jena/abac/labels/RocksDBHelper.java

```diff
@@ -0,0 +1,139 @@
+package io.telicent.jena.abac.labels;
+
+import org.rocksdb.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static io.telicent.jena.abac.labels.Labels.LOG;
+
+public class RocksDBHelper {
+
+    private final static String CREATE_MESSAGE = "create of RocksDB label store";
```

Comment:
Utterly pedantic and I know predates your work.
```suggestion
    private final static String CREATE_MESSAGE = "creation of RocksDB label store";
```